packages/core/src/components/discord-modal/DiscordModal.ts

```diff
@@ -458,9 +458,9 @@ export class DiscordModal extends LitElement implements LightTheme {
 
 		const expanderT = this.shadowRoot?.querySelector('slot');
 
-		const slotedItems = expanderT?.assignedElements();
+		const slotedItems = expanderT!.assignedElements();
```

Comment:
Do not use `!.` (non null assertions), Unlike `?.` (optional chaining) they are removed when compiled and can lead to unexpected errors being thrown for JavaScript users.